templates/archive-course.php

```diff
@@ -13,21 +13,23 @@
 
 <?php get_sensei_header(); ?>
 
-	<?php
-
-		/**
-		 * This action before course archive loop. This hook fires within the archive-course.php
-		 * It fires even if the current archive has no posts.
-		 *
-		 * @since 1.9.0
-		 *
-		 * @hooked Sensei_Course::course_archive_sorting 20
-		 * @hooked Sensei_Course::course_archive_filters 20
-		 * @hooked Sensei_Templates::deprecated_archive_hook 80
-		 */
-		do_action( 'sensei_archive_before_course_loop' );
-
-	?>
+	<section class="sensei-archive-controls">
```

Comment:
The `<section>` element doesn't feel like the proper semantic choice here as it contains the filters and the course header. It seems as though it's being used as a generic container, so `<div>` feels more appropriate.

Also, since we're changing this template, we'll need to bump the version number up top (to 3.0.0 in this case). Other sites can override these templates, so we try to change them sparingly since making changes sometimes means we break their overrides until they update their own templates. In this case the change seems fairly benign, so we should be OK. 🤞

Reply:
True, changed that to div and bumped the version.